### ISSUE TITLE
Absent Neutrons in Neutron Emission of 11-Be

### ISSUE THREAD

**Issue by scshadrick** [GRIFFINCollaboration/detectorSimulations_v10#61]
I'm simulating beta-delayed neutron emission of 11-Li -> 11-Be -> 10-Be on the newest detector_sims and geant4.10.04.p02.  I have updated my data files accordingly.

When I feed into an excited state of 11-Be from 11-Li, I get the expected neutron emissions from states 2654 to 5255 keV.  However, I do not see any neutrons for states 7030, 8020, or 8813 keV in 11-Be, and I get a runtime error of "No decay channels available for 10590 keV".  

Is there an energy cut in the newer detector_sims code in the physics lists I'm missing? or could this be a geant4 problem with the current edition? 

Relevant z3.a11 file
 11LI (  8.75 MS  )
  Excitation  flag   Halflife  Mode    Daughter Ex flag   Intensity          Q
P            0  -      0.00875
                              BetaMinus            0               1
                              BetaMinus       320.04  -       7.7     20231.05
                              BetaMinus         2654  -       17.0     17897.09
                             BetaMinus         3400  -       0.9     17151.09
                              BetaMinus         3889  -       22.7     16662.09
                              BetaMinus         3955  -       6.8     16596.09
                              BetaMinus         5255  -       2.4     15296.09
                              BetaMinus         7030  -      0.86     13521.09
                             BetaMinus         8020  -       15.5     12531.09
                              BetaMinus         8813  -       8.9     11738.09
                             BetaMinus        10590  -       7.8     9961.088
                             BetaMinus        16300  -     0.052656     4251.088
                              BetaMinus        18190  -      0.60335     2361.088`

Relevant z4.a11 file
P           10590 -    1.e-25
                         Neutron              0            1
                         Neutron           9270 -      92.190       816
                         Neutron           3368 -       5.250       6718
                         Neutron            0   -     2.560       10086
P            8813 -    1.e-25     
                         Neutron              0            1
                         Neutron         7371  -      75.281     938
                         Neutron         6263  -      24.720     2046
P            8020 -    1.e-25
                         Neutron              0            1
                         Neutron         6179  -     10.828     1347
                         Neutron         5958  -     61.783     1532
                         Neutron         3368  -     22.293     4158
                         Neutron            0  -    5.096       7516
P            7030 -    1.e-25
                         Neutron              0            1
                         Neutron           3368 -      100       3158
P            5255 -    1.e-25
                         Neutron              0            1
                         Neutron           3368 -         100       1368
P            3955 -    1.e-25
                         Neutron              0            1
                         Neutron           3368 -         77.941        88
                         Neutron           0    -      22.059        3451
P            3889 -    1.e-25
                         Neutron              0            1
                         Neutron           3368  -        61.674        18
                         Neutron           0     -    38.326        3385
P            3400 -    1.e-25 
                         Neutron              0            1
                         Neutron              0   -       100        2896
P            2654 -    1.e-25
                         Neutron              0            1
                         Neutron              0   -       100        2150

**Comment by VinzenzBildstein** [comment]
I don't think that there is any energy cut in the physics lists, at least not one that wasn't there in earlier versions. Do I understand right that these files were working with an older version of the code?

**Comment by VinzenzBildstein** [comment]
Has anything changed in the file format for the data files between the geant4 versions? I think at this point it is hard to tell whether this is an issue with geant4 or our simulation, though I don't think we've changed anything in the simulation itself that would break this.

**Comment by scshadrick** [comment]
The only change to the data files has been the addition of the transition flag between the excited state energy and half-life in the branch initialization and between the daughter excited state energy and decay probability in the transition info (which I had set to - for all the states, if I read correctly means it's not a variable transition.  I'm not sure if this is correct either, but I tried a few different flags with no discernible effect).  Example for newer version:

P    8020     -     1.e-25
   Neutron    0     1
   Neutron    6179    -    10.828    1347
   Neutron    5958    -    61.783    1532
   Neutron    3368    -    22.293    4158
   Neutron    0          -    5.096      7516

compared to the older version:

P    8020         1.e-25
   Neutron    0     1
   Neutron    6179       10.828    1347
   Neutron    5958       61.783    1532
   Neutron    3368       22.293    4158
   Neutron    0             5.096      7516

I don't think it's anything with the simulation itself; all I'd changed was defining the particle iterator in some functions and the fermi model definition like I said before, but maybe these changes play strangely with the newer geant4? I'm not sure. And it's especially weird to me that it only affects certain decays. 

I have been using grep through the source code of the newer geant4 for energy cuts and transition info to see if I can find anything, but to no avail.
